README.md

```diff
@@ -4,6 +4,59 @@
 
 This application connects tags from a Flexy to an OSIsoft PI dataserver.
 
+## Getting Started
+
+### Configuration 
+
+All user configuration is done through the ConnectorConfig.json file.  This file must be modified to match your configuration.
+
+#### Example ConnectorConfig.json
+```
+{
+   "ServerConfig":{
+      "IP":"192.168.0.124",
+      "WebID":"s0U1IjG6kMOEW7mxyHCuX2mAUEktU0VSVkVSLVBD",
+      "Credentials":"UEktU2VydmVyOk1hbmNoZXN0ZXIxMjMh"
+   },
+   "eWONConfig":{
+      "CertificatePath":"/usr/Certificates"
+   },
+   "AppConfig":{
+      "CycleTimeMs":1000
+   },
+   "TagList":["ExampleTag1", "ExampleTag2", "ExampleTag3", "ExampleTag4"]
+}
+```
+#### ServerConfig
+
+IP - IP address of your OSIsoft PI server
+
+WebID - WebID of your OSIsoft PI dataserver
+
+#### Credentials - Base64 encoded user credentials for basic authentication 
+
+To generate your Base64 encoded user credentials visit https://www.base64encode.org/ and encode "username:password"
+
```

Comment:
Maybe an example to help. Like:

For example, if your username is 'username' and your password is 'password' you would encode "username:password" and should get "dXNlcm5hbWU6cGFzc3dvcmQ="